ssh/tailssh/tailssh.go

```diff
@@ -1154,6 +1155,10 @@ func (ss *sshSession) run() {
 	return
 }
 
+// recordSSHToLocalDisk is a temporary dev knob to test the SSH recording
+// functionality and support off-node streaming.
```

Comment:
> and support off-node streaming

This knob is just for enabling local storage, no?

Reply:
yep, updated docs